src/styles/utils/_each.scss

```diff
@@ -0,0 +1,17 @@
+/* variables for this loop from _variables.scss file
+
+$daysOfWeek:
+  "mon" 0,
+  "tue" ($boxWidth),
+  "wed" ($boxWidth * 2),
+  "thu" ($boxWidth * 3),
+  "fri" ($boxWidth * 4),
+  "sat" ($boxWidth * 5),
+  "sun" ($boxWidth * 6);
+  */
+
+@each $day, $dayShift in $daysOfWeek {
+  .calendar--start-day--#{$day} :nth-child(1) {
+    margin-left: #{$dayShift};
+  }
+}
```

Comment:
why did you add loops in separate files? its not a mixin or function etc.

Reply:
because I wanted them separately. I thought it is by the rules. But OK, I will separate all Loops in the file